kolena/_utils/validators.py

```diff
@@ -13,16 +13,16 @@
 # limitations under the License.
 from typing import Optional
 
-from pydantic import Extra
+from pydantic import ConfigDict
 
 from kolena._utils.consts import FieldName
 
 
-# Pydantic configuration for dataclasses and @validate_arguments decorators
-class ValidatorConfig:
-    arbitrary_types_allowed = True
-    smart_union = True
```

Comment:
I see that this was removed from Pydantic -- do they provide an alternative? I remember we added this because numerics were being coerced into strings because of union types.

Reply:
According to https://github.com/pydantic/pydantic/discussions/7264, `smart_union=True` is now the default behavior:

> By default, v2 has the "smart union" behaviour. If you want to keep the behaviour of v1 you can use ...